async_fuse/src/byte_slice.rs

```diff
@@ -16,31 +19,36 @@ unsafe fn transmute_ref_unchecked<T: FuseAbiData>(bytes: &[u8]) -> &T {
 /// + ensure that `bytes.as_ptr()` is well-aligned for T.
 /// + ensure that `bytes.len()` is a multiple of `size_of::<T>()`.
 unsafe fn transmute_slice_unchecked<T: FuseAbiData>(bytes: &[u8]) -> &[T] {
-    let len = bytes.len() / mem::size_of::<T>();
+    let len = bytes.len().overflow_div(mem::size_of::<T>());
```

Comment:
The value of `mem::size_of::<T>()` is always non-negative.
And the unsafe marker `FuseAbiData` ensures that `T` is not ZST.
So the unsigned integer division here is impossible to overflow.
Consider using `wrapping_div`.

Reply:
Let's make change to utilities later, instead of change here.
Currently we are in debug stage, it prefers fast fail, so I changed utilities to use `assert` instead of `debug_assert` when overflow occurs.
Later when the code is stabilized, we can change utilities to use `wrapping_*` instead of `overflowing_*`.
I've added an [issue](https://github.com/datenlord/utilities/issues/5) to track this.